src/components/Header.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
```

Comment:
create an index file in the components folder and export like i've shown you on other index files below